trailpanion/src/data/store.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit'
+import mapsReducer from './mapsSlice'
+
+export const store = configureStore({
+  reducer: {
+    maps: mapsReducer
```

Comment:
We currently just have a maps reducer but we can add more here as your app expands